src/qibosoq/programs/sweepers.py

```diff
@@ -94,6 +94,9 @@ def add_sweep_info(self, sweeper: Sweeper):
                     max_gain = int(self.soccfg["gens"][gen_ch]["maxv"])
                     starts = (sweeper.starts * max_gain).astype(int)
                     stops = (sweeper.stops * max_gain).astype(int)
+                elif sweeper.parameters[idx] is Parameter.START:
+                    # TODO tell qick of the mismatch (t != time)
```

Comment:
If I understood correctly the mismatch is on their side, since `self.get_ren_reg` should be a function of `NDAveragerProgram`.

Reply:
Yes I think so